README.md

```diff
@@ -34,9 +34,16 @@ jobs:
       - uses: julia-actions/setup-julia@latest
       - name: VersionVigilante.main
         id: versionvigilante_main
+        # BEGIN: Uncomment these lines if this is a private repository
+        # run: |
+        #   julia -e 'using Pkg; Pkg.add("VersionVigilante")'
+        #   julia -e 'using VersionVigilante; VersionVigilante.main("https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/${{ github.repository }}")'
+        # END: Uncomment these lines if this is a private repository
+        # BEGIN: Comment out these lines if this is NOT a private repository
```

Comment:
Should this not be "Comment out these lines if this **IS** a private repository"? If I follow the instructions in a public repo, I'd leave the lines above commented out and then *also* comment these lines, or am I misunderstanding something?

What do you think about:

```yaml
        run: |
          julia -e 'using Pkg; Pkg.add("VersionVigilante")'

          # Public repos:
          julia -e 'using VersionVigilante; VersionVigilante.main("https://github.com/${{ github.repository }}")'

          # Private repos:
          # julia -e 'using VersionVigilante; VersionVigilante.main("https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/${{ github.repository }}")'
```

I think it's a bit cleaner to not duplicate the whole `run` block but only the line that actually changes.